### PR TITLE
Bug 1880357: Bump k8s.io packages

### PR DESCRIPTION
**Pull request by coreydaley-redhat**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -30,38 +30,38 @@ require (
 	github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5 // indirect
 	github.com/xeipuuv/gojsonschema v1.2.0 // indirect
 	go.uber.org/zap v1.11.0 // indirect
-	k8s.io/api v0.18.0
-	k8s.io/apimachinery v0.18.0
-	k8s.io/apiserver v0.18.0
-	k8s.io/client-go v0.18.0
-	k8s.io/component-base v0.18.0
+	k8s.io/api v0.18.8
```
Comment (otaviof) [inline go.mod:33]: Should it be based on `v0.18.9`? The latest version on the 0.18 release

Comment (adambkaplan) [reply go.mod:33]: I think k8s cut 0.18.9 very recently. 0.18.8 is sufficient for this bugfix.

**Review by adambkaplan — APPROVED** [review]
/lgtm

**Pull request merged**